projects/sunbird-player-sdk/src/lib/core/components/end-page/end-page.component.html

```diff
@@ -33,8 +33,8 @@
           <stop stop-color="#1D6349" offset="100%"></stop>
         </linearGradient>
         <text id="text-8" x="55" y="16" text-anchor="middle" fill="#FFFFFE" *ngIf="outcome">
-          <tspan font-size="12" font-weight="400" font-family="Noto Sans, NotoSans-Bold">{{outcomeLabel}}</tspan>
-          <tspan font-size="18" font-family="NotoSans-Bold, Noto Sans">{{outcome}}</tspan>
+          <tspan font-size="12" font-weight="400" font-family="Times New Roman">{{outcomeLabel}}</tspan>
```

Comment:
Don't replace the font but add `Times New Roman` at the beginning to take priority